### DISCUSSION TITLE
Change of ip address

### DISCUSSION THREAD

**Discussion by norbergmats** [wlcrs/huawei_solar#359] [General]
Hi, if I need to change the IP address of the sDongle. Can I change the IP in the integration without re-installing it in HA by editing some configuration file ?

**Comment by nicolasdew** [comment]
I was going to post the exact same question. 
For info i'm running HA from a docker image, and i'm not sure how to access the configuration files.
Any answer would be greatly appreciated.

**Comment by chalkstorm** [comment]
Any update on this? Save uninstall and reinstall? Did use a static IP but have some issues with my SDongle and stable connectivity

**Comment by g3r0nim00** [comment]
I've noticed you don't loose any data when removing the inverter and re-adding it, although for me this was always using the same IP...
Maybe if you remove it and add it again with the new IP you won't loose any data?

**Comment by chalkstorm** [comment]
Correct.  I've done it several times now.  Only odd thing is some entities appear as disabled (internal temperature) so have to go in and enable it.

**Comment by grimmaldus** [comment]
So still no reconfiguration option available ?
